source/utils/environment.js

```diff
@@ -22,3 +24,23 @@ export function getServicesBaseURL() {
 export function getEventLoggerBase() {
     return `${getServicesBaseURL()}event-logger`;
 }
+
+export function isSandbox() {
+    if (!isBrowser() || isProduction()) {
```

Comment:
I think we need to remove the check for production here. We will probably want to include the check for `isSandbox ` inside of `isProduction`. With this code, we will end up with a circular dependency causing an infinite loop.